src/SFA.DAS.Reservations.Functions.LegalEntities.UnitTests/WhenSignedAgreementEventTriggered.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Reservations.Domain.AccountLegalEntities;
+using SFA.DAS.Reservations.Domain.Infrastructure;
+
+using SFA.DAS.Reservations.Infrastructure;
+
+namespace SFA.DAS.Reservations.Functions.LegalEntities.UnitTests
+{
+    public class WhenSignedAgreementEventTriggered
+    {
+        [Test]
+        public async Task ThenQueueMessageWillBeCreated()
```

Comment:
these test names could use underscores to keep consistent, but no big deal for now. 
there's a few other files with same non-underscored test names.